flybywiresim/a32nx
# fix(efb): state persistence update for landing perf

Fixes #4575, Supports PR https://github.com/flybywiresim/a32nx/pull/4026

## Summary of Changes
- updated landing performance with reducer to persist state in session
- updated onchange function for select component to accept boolean as arg
- fix incorrect map theme on dashboard
- TODO: context and reducer for performance to be refactored in checklists pr

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): antiso

## Testing instructions

- View Landing Performance view on EFB, change some values
- View another view on the EFB
- Navigate back to the Landing Performance view and observe values persisted


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
